### ISSUE TITLE
Contrast problems with inline code elements

### ISSUE THREAD

**Issue by bmuenzenmeyer** [pattern-lab/website#124]
See https://patternlab.io/docs/advanced-starterkits.html#psa

![image](https://user-images.githubusercontent.com/298435/68678344-6550a680-0523-11ea-95ce-15554e0d6afe.png)

This may be cleaned up with the redesign, but I wanted to capture it nonetheless

**Comment by bradfrost** [comment]
@bmuenzenmeyer This will definitely be cleaned up with the redesign!

**Comment by sghoweri** [comment]
Oh good - glad I wasn't the only one who noticed this!

**Comment by andrewcodes404** [comment]
When is the re-design happening because as a new pattern lab user the docs are impossible to read

**Comment by bradfrost** [comment]
@andrewcodes404 the redesign is underway, and you can check it out here: https://pattern-lab-eleventy.netlify.com/docs/editing-pattern-lab-source-files/

Hoping to get things live soon.

**Comment by dartanian300** [comment]
@bradfrost 

Not to be _that _guy__, but do you have any idea when that might be? We're looking to start work on a new pattern library and I want to make sure I give Pattern Lab a fair shake (it's fairly unusable in the website's current state).

**Comment by bradfrost** [comment]
@dartanian300 thanks for the nudge, and I'm embarrassed this hung out in this state for so long. @frostyweather just took care of the issue so inline code is actually legible.

This of course is a stopgap, and I'd like to get the new site launched right quick. @sghoweri @bmuenzenmeyer, would one of you be able to help get the new site into the monorepo and launched?

**Comment by bmuenzenmeyer** [comment]
@bradfrost yeah this sounds like a plan! can you share with me the repo it currently is in?

**Comment by bradfrost** [comment]
@bmuenzenmeyer cool just invited you to that repo.

**Comment by sghoweri** [comment]
lol @bmuenzenmeyer just beat me to this. 

Guys, lemme know if either of you could use some help migrating the new site over or wiring things up!
